src/plugins/.#parametric-curve.lisp

```diff
@@ -0,0 +1 @@
+kaveh@Kavehs-MacBook-Pro.local.458
```

Comment:
@kaveh808 you might have an accidental emacs temp file committed here.

Reply:
@cmack Thanks for catching this. I think I fixed it.